src/chunk_processors.py

```diff
@@ -15,6 +15,30 @@
 logger = getLogger(__name__)
 
 
+def filter_and_warn_for_unknown_types(types: list[str]) -> list[str]:
+    """
+    Filter out unknown types from a list of types.
+
+    If the type is unknown, log a warning and remove it from the list.
+    """
+
+    types_to_remove: list[str] = []
```

Comment:
micro nit picking: if i'm reading it correctly, i think that if this is passed a long list of `types` with repetitions, we'll get multiple warnings for the same value. it doesn't look like `filter_and_warn_for_unknown_types` is being passed any long lists like that right now, but maybe comparing `set(types)` with `set([type.value for type in BlockType])` would give us the same behaviour here, while guarding against that possibility in the future?

Reply:
good catch! whoops